pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/FileUploadService.java

```diff
@@ -84,8 +84,10 @@ public static FileUploadService fromConfig(Config cfg) {
         long removalExpireTime = cfg.getLong(ConfigFile.FileUploads.REMOVAL_EXPIRE_TIME);
         TimeUnit removalExpireUnit = TimeUnit.valueOf(cfg.getString(ConfigFile.FileUploads.REMOVAL_EXPIRE_UNIT));
 
+        var apiKey = cfg.getString(ConfigFile.SENDGRID_API_KEY);
+        var connectProxy = ConfigUtil.getStrIfPresent(cfg, ConfigFile.Sendgrid.PROXY);
         return new FileUploadService(
-                new SendGridClient(cfg.getString(ConfigFile.SENDGRID_API_KEY)),
+                new SendGridClient(apiKey, connectProxy),
```

Comment:
I checked vault for all our environments. We proxy address in dev, staging, and prod but not in test.
Does not to be a problem though.

Reply:
In that case, this will just fallback to existing behavior (which is fine- test does not need the proxy)